### DISCUSSION TITLE
Built in calendar?

### DISCUSSION THREAD

**Discussion by murph12F** [neomutt/neomutt#4130] [Development]
Would be great to have a built in calendar(if not already there and I didn't see it) to fully replace a gui mail client, that s the only thing that keeps me from deleting once and for all thunderbird. 
In the meanwhile ant workaround to also manage invitations and calendar? 

Thx a lot :)

**Comment by mschilli87** [comment]
I for one believe in do-one thing and do it well. I am using neomutt for mail, (i)khal for calendaring, khard for contacts, vdirsyncer to sync my calendars and contacts between devices and screen to multiplex my terminal to combine all the above into an interface I like. I think there is enough complexity in neomutt as it stands.

**Reply by murph12F** [reply]
and how the integration with neomutt and ikhal works? is it automatic, in the sense that if i receive an invite i get it on my calendar automatically? how can i set it up, or if there is any doc to do that? 
thx for the response

**Reply by mschilli87** [reply]
I configured a MIME-time for calendar attachments to import them to khal. If you can't find a guide, I might link some docs here later. Right now I don't have them at hand.

**Comment by flatcap** [comment]
Long ago, when NeoMutt started, I had ideas about a more Office-y solution.
That was before I realised the dire state of the code (it's much better now :-)

> I for one believe in do-one thing and do it well.
> khal for calendaring, khard for contacts,

Yeah, I agree, but there's room for improvement.

Calendars and contacts are essential.
We need a seamless way to integrate your tools of choice.

But for that to happen, we need people to help.

- Which tools are you using?
- How do you integrate them?
- What changes to NeoMutt would make things simpler?  or more versatile?

**Reply by murph12F** [reply]
Hello thanks for your time,
As of now I am not using neomutt cause it doesn't have a built in calendar, and i dont want to have different tools for those thing (I do believe mail and calendar at this point should be on the same tool) so I can't tell you what is my setup or what I d do better, I ve googled around and I think that the most used might be neomutt integrated with khal, but unfortunately I can't really provide more info than this or my personal experience bout it.

**Comment by flatcap** [comment]
@mschilli87 
Could we ship your config for khal/khard in NeoMutt's `contrib/`?
We'd need some simple docs explaining how to set use each tool from NeoMutt.

**Reply by mschilli87** [reply]
@flatcap: In principle yes, but I'd need to dig through them myself first. They are a crazy collections cluttered together over years and while they work for me, I am sure there are things that could (and should) be improved, cleaned up before sharing them as a sort of recommendation. Specifically, I'd like to have clean integration with gpg and/or pass for authentication with the variuous servers involved. I won't have time in 2023 I am afraid but 'early' next year I could tackle this. Feel free to ping me here if I procrastiante too long on it. 😉

**Reply by flatcap** [reply]
Great!  Thanks!

> crazy collections cluttered together over years

hehe, your config too ;-)

Feel free to dump ideas and raw config in this discussion.

> early' next year ... free to ping me here if I procrastiante too long

No rush, will do.
